### PR TITLE
Add support for alternative file types

### PR DESCRIPTION
**Pull request by ssaraswati**
Support generating kroki diagrams as types other than svg to support rendering diagrams in more scenarios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ plugins:
 * `EnableMermaid` - Enable Mermaid, default: True
 * `HttpMethod` - Http method to use (`GET` or `POST`), default: `GET` (Note: you have to enable `DownloadImages` if you want to use `POST`!)
 * `DownloadImages` - Download diagrams from kroki as static assets instead of just creating kroki links, default: False
-* `DownloadDir` - The asset directory to place downloaded svg images in, default: images/kroki_generated
+* `DownloadDir` - The asset directory to place downloaded images in, default: images/kroki_generated
+* `FileType` - File type you want to use, default: svg, (Note: not all file formats works with all diagram types)
```
Comment (b-bittner) [inline README.md:30]: Maybe add a reference URL, eg. https://kroki.io/#support

**Comment by b-bittner** [comment]
Hello @ssaraswati ,
thanks for your PR, every help to make this project better ist welcome.
What I miss is an option to set the filetype per diagram, for example what happens, when I set it in the config file to png and a diagramm-type doesn't offer png. Or when I just what to have one in PDF. Maybe also a fallback to svg whould be nice.
Also, for some reasons, all types wich do not support png, aren't working anymore, even when svg ist configured.
Can you try to add these changes...

**Comment by ssaraswati** [comment]
Hi @b-bittner I have pushed up a change that I think addresses the additional functionality you were after.

You can now supply a list of file types you wish to generate with the first matching filetype for a diagram being what's used. Including SVG (or other types) in this list will let it be used as a fallback for any diagrams that don't support a first preference. 

I have also added a FileTypeOverrides that lets you set the specific format per diagram.

The bug with PNG being the only generated image has also been fixed. Please let me know if you have any other suggestions/improvements I can make

**Comment by b-bittner** [comment]
Added some minor formating changes and commited it in a new brach/pr.
Thanks for your changes, they are really welcome!

**Pull request closed without merging**